### PR TITLE
Mars 317 kiva context borrower profile

### PR DESCRIPTION
**Pull request by christian14b**
This experiment has two versions for loans that belongs to the following categories (these are ranked too):

1. Eco-friendly
2. Education
3. Arts
4. Agriculture
5. Women
6. Kiva US

**A** version adds `loan impact` section and chunk text in `story`  and `more about this loan` sections. Also, adds a new `vetted by` section.

image
image

**B** version displays the expected repayment date 
image

### CODE REVIEW

**Review by mcstover — CHANGES_REQUESTED** [review]
@christian14b We are in the process of trying to reduce the initial page load for the Borrower Profile so should not be adding any additional prefetched data. As noted in other discussions we are actively auditing and removing unnecessary data from the prefetch over the next couple sprints.

Please move any new data needed for your experiments to it's own query that runs in the mounted hook - or created if wrapped with a `!this.$isServer` so it only runs in the browser. You will need to create placeholders where the content will eventually show up to prevent significant content jumps for at least the Loan Story additions.

**Comment by christian14b** [comment]
@mcstover Let me know if that change fix the problem or if there's something else to change regarding prefetched data

File: src/components/BorrowerProfile/LoanStory.vue [hunk 1]
```diff
@@ -96,13 +131,30 @@ export default {
...
+						country {
+							isoCode
+						}
+					}
+					tags
+					gender
+					sector {
+						id
+						name
+					}
```
Comment (mcstover) [inline src/components/BorrowerProfile/LoanStory.vue:144]: This is a prefetched query. Can these items be placed in a new query activated by the mounted hook instead?

File: src/components/BorrowerProfile/LoanStory.vue [hunk 2]
```diff
@@ -96,13 +131,30 @@ export default {
...
 							id
 							bylineName
 							showName
 						}
+						partner {
+							id
+							countries {
+								name
+							}
+						}
```
Comment (mcstover) [inline src/components/BorrowerProfile/LoanStory.vue:157]: Same here...

File: src/components/BorrowerProfile/LoanStory.vue (hunk 1 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LoanStory.vue:144]: Looks like you've already duplicated them in mounted so maybe these can just be removed...

Comment (christian14b) [reply src/components/BorrowerProfile/LoanStory.vue:144]: Thought I removed them, thanks!

**Review by mcstover — APPROVED** [review]
Thank you @christian14b This exp has some really neat informational addtions to the page!

**Review by dyersituations — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
Looks good!

**Pull request merged**